Library/Homebrew/livecheck.rb

```diff
@@ -87,6 +88,23 @@ def regex(pattern = T.unsafe(nil))
     end
   end
 
+  # Sets the `@throttle` instance variable to the provided `Integer` or returns
+  # the `@throttle` instance variable when no argument is provided.
+  sig {
+    params(
+      # Throttle rate of version patch number to use for bumpable versions.
+      rate: T.nilable(Integer),
```

Comment:
This is ` T.nilable(Integer)` because this method is also used to get the `@throttle` value (i.e., we use `throttle 10` in the DSL and `formula.livecheck.throttle` to obtain the value).

Reply:
That still works if `rate: Integer`, because it is only checked when passed in from the outside.

Reply:
I.e., currently all of these are allowed `throttle 1`, `throttle nil` and `throttle`, by specifying `rate: Integer`, we disallow passing `nil` explicitly.

Reply:
Makes sense. Thanks for clarifying. I pushed a commit to incorporate this change.